mozilla-lockbox/lockbox-android
# Don't spell check the detail view items

Fixes a visual bug that shows a red underline on the hostname and username fields

## Testing and Review Notes

- open detail view of "accounts.firefox.com" for `firefoxlockbox@gmail.com` test account
- tap on the hostname or username fields
- tap on the copy username button
- observe the red underline does not appear

## Screenshots or Videos

Before:
<img width="275" alt="screen shot 2018-12-17 at 4 05 19 pm" src="https://user-images.githubusercontent.com/49511/50121541-49128f80-0216-11e9-93b2-a17c3d685cd6.png">

After:
<img width="273" alt="screen shot 2018-12-17 at 4 08 51 pm" src="https://user-images.githubusercontent.com/49511/50121542-49128f80-0216-11e9-82ea-e3e646fd2c9c.png">

I believe unrelated to this two line change but I think the nav bar in detail view disappeared on `master` recently? Or maybe thats just my messy local environment?
